xrpl/models/requests/amm_info.py

```diff
@@ -3,9 +3,10 @@
 
 from dataclasses import dataclass, field
 
+from pyparsing import Optional
```

Comment:
The tests look to be failing due to using `pyparsing` and it not being added to the poetry config.  I think `typing` should be used instead.